torch/library.py

```diff
@@ -32,6 +33,8 @@
     "register_vmap",
     "get_ctx",
     "custom_op",
+    "triton_op",
```

Comment:
I can't find the doc being rendered for these on the website. Am I missing something ?

Reply:
Weird, let me dig into it

Reply:
RIP my browser cached the artifacts. If your browser does too then I'd recommend opening the docs page in incognito mode lol

Reply:
This is weird because putting triton_op in the search doesn't bring it up.
cc @svekars is that due to the new search? This sounds bad?

Reply:
> This is weird because putting triton_op in the search doesn't bring it up. cc @svekars is that due to the new search? This sounds bad?
The search goes to stable docs most likely (the indexed pages). I will add a PR to enable old search in docs.preview.

Reply:
Thanks!